app/routes.js

```diff
@@ -30,13 +30,27 @@ const addContext = ({ context, req, csrfToken }) => ({
 
 export const routes = (authProvider, sessionManager) => {
   const router = express.Router();
+  const a = new Date();
```

Comment:
Do we need to use milliseconds? 
The config value can be passed in as date for comparison.

Also do we not need tests for the code or have they been added in another check in?

Reply:
I don't think there is any problem in using a timestamp, it is very easy to get a hold of and compare and I think it avoids potential problem with timezones.

Reply:
Timestamp is ok then but unit tests should be added